frontend/src/store/Books/BooksReducer.ts

```diff
@@ -6,9 +6,9 @@ import {
 	DispatchBookActions,
 } from "./BookTypes";
 
-interface BooksState {
+export interface BooksState {
 	loading: boolean;
-	books: Books | null;
+	books: Books[] | null;
```

Comment:
Don't put null, leave it as Books[]. Empty array for initialState. InitialState books: []